plugin/githubinator.vim

```diff
@@ -89,13 +87,18 @@ function! s:github_copy_url()
         return
     endif
 
-    echom 'Githubinator: URL copied to clipboard.'
+    echom 'Githubinator: URL copied to clipboard: ' . l:final_url
 endfunction
 
-vnoremap <silent> <Plug>(githubinator-open) :<C-U>call <SID>github_open_url()<CR>
-vnoremap <silent> <Plug>(githubinator-copy) :<C-U>call <SID>github_copy_url()<CR>
+vnoremap <silent> <Plug>(githubinator-open) :<C-U>call <SID>github_open_url('v')<CR>
+vnoremap <silent> <Plug>(githubinator-copy) :<C-U>call <SID>github_copy_url('v')<CR>
+nnoremap <silent> <Plug>(githubinator-open) :<C-U>call <SID>github_open_url('n')<CR>
+nnoremap <silent> <Plug>(githubinator-copy) :<C-U>call <SID>github_copy_url('n')<CR>
```

Comment:
The current move is given to distinguish normal mode and visual mode to get the line numbers.
Please give me more clever way if you found 😃